events/test_routes.py

```diff
@@ -0,0 +1,82 @@
+import unittest
```

Comment:
Add module docstring and Apache 2.0 license.

Reply:
I think a lot of my files are missing the license right now, I'll add it to all of them in another PR

Reply:
Added docstring in d8c42af